src/App.vue

```diff
@@ -26,6 +31,10 @@ export default {
   },
   computed: {
     ...mapGetters(['settings', 'tipSortBy']),
+    isSupportedBrowser() {
+      const browser = detect();
+      return !IS_MOBILE_DEVICE && (browser && supportedBrowsers.includes(browser.name));
```

Comment:
I think mobile should be "supported". if "yes" should revert the check and use ||

Reply:
on mobile we use deep links so i am not sure if it should be supported.
@mradkov ?

Reply:
On mobile we do not need to show this.